trilby-cli/src/Trilby/HNix.hs

```diff
@@ -52,3 +53,18 @@ canonicalSet s = error $ "canonicalSet bottom: " <> show s
 
 listToSet :: (ToExpr a) => (a -> NAttrPath NExpr) -> [a] -> NExpr
 listToSet f xs = Fix $ NSet NonRecursive $ (\x -> f x ~: toExpr x) <$> xs
+
+data FlakeRef = FlakeRef
+    { url :: Text
+    , output :: Maybe Text
+    }
```

Comment:
@vkleen are these the correct terms of art? I worry it might be `<flakeRef>#<output>`, but then I don't know what the whole thing is.

Reply:
I think so. At least, I can't think of anything better than `output`.

Reply:
Actually, the nix manual uses `flakeRef` and `attrPath` in https://nixos.org/manual/nix/unstable/command-ref/new-cli/nix#installables. I'm not convinced :laughing:

Reply:
Let’s name these

```haskell
data FlakeOutput = FlakeOutput
    { flakeRef :: Text
    , attrPath :: Maybe Text
    }
```